src/clj/rems/home.clj

```diff
@@ -40,11 +48,15 @@
   (GET "/" []
     (layout/home-page))
 
+  ;; TODO should these redirects have swagger documentation?
+
   (GET "/accept-invitation" [token]
     (redirect (str "/application/accept-invitation/" token)))
 
-  (GET "/apply-for" [resource]
-    (apply-for-resource resource))
+  (GET "/apply-for" [resource] ;; can specify multiple resources
+    (if (vector? resource)
```

Comment:
Could this break if compojure decides to return just a seq?

Reply:
The ring wiki says it's a vector. https://github.com/ring-clojure/ring/wiki/Parameters